modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/deployers/terraform/terraformlocal/TerraformLocalExecutor.java

```diff
@@ -184,15 +187,40 @@ public void destroy() {
         }
     }
 
+    private boolean isFileLocked(String filePath) {
```

Comment:
duplicate piece of code. Please move it to a common place and use it in both deployers.